examples/dcgan/dcgan.gop

```diff
@@ -28,13 +30,108 @@ var (
 	cuda       = flag.Bool("cuda", false, "enables cuda")
 	dryRun     = flag.Bool("dry-run", false, "check a single training cycle works")
 	ngpu       = flag.Int("ngpu", 1, "number of GPUs to use")
-	netG       = flag.String("netG", "", "path to netG (to continue training)")
-	netD       = flag.String("netD", "", "path to netD (to continue training)")
+	pathG       = flag.String("pathG", "", "path to netG (to continue training)")
```

Comment:
Probably better to keep the naming consistent with the Python version?

Reply:
The network name is also `netG`. So, I rename it to `netGPath`.